dev-manual/api/api-reference-archivematica.rst

```diff
@@ -224,9 +224,25 @@ Hide
              delete/**
 ===========  =================================   ==============================
 
+Request body parameters:
+
+=================   ================================================================
+
+``transfer_uuid``   UUID OF THE TRANSFER.
+
+=================   ================================================================
```

Comment:
This table doesn't need to be so wide, right? Can we adjust it to the minimum necessary? Same with the table below.

Reply:
sure